### PR TITLE
add change for endpoint get auth/me

### PR DESCRIPTION
**Pull request by DanteIturri**
add endpoint in UserController
add inplement UserService 
add method in UserService Implement

### CODE REVIEW

File: src/main/java/com/alkemy/ong/controller/UserController.java [hunk 1]
```diff
@@ -35,6 +36,10 @@
         List<UserDTO> users = userService.getAllUser();
         return ResponseEntity.ok().body(users);
     }
+    @GetMapping("auth/me")
```
Comment (anavduo) [inline src/main/java/com/alkemy/ong/controller/UserController.java:39]: is this method in the UserAuthController? should the route be only "/me"? I am not sure if it is the same

Comment (DanteIturri) [reply src/main/java/com/alkemy/ong/controller/UserController.java:39]: this method shows the data after authenticating and the endpoints are public of UserAuthController

File: src/main/java/com/alkemy/ong/service/impl/UserServiceImpl.java [hunk 2]
```diff
@@ -20,6 +22,9 @@
     @Autowired
     UserMapper userMapper;
```
[the remark this answers is not in the document]
Comment (DanteIturri) [reply src/main/java/com/alkemy/ong/service/impl/UserServiceImpl.java:23]: the userMapper was already just use

File: src/main/java/com/alkemy/ong/controller/UserController.java (hunk 1 above)
Comment (anavduo) [reply src/main/java/com/alkemy/ong/controller/UserController.java:39]: got it thanks!

**Review by juan1977lopez — APPROVED** [review]
approved

**Pull request merged**